projects/js-packages/publicize-components/src/components/connection-icon/style.scss

```diff
@@ -33,4 +43,10 @@
 			}
 		}
 	}
+
+	.jetpack-publicize-gutenberg-social-icon,
+	.jetpack-publicize-connection-label-copy {
+		display: inline-block;
+		vertical-align: middle;
+	}
```

Comment:
Definitely not this PR, but I wonder if we should import this CSS similar to how we do it in other places (`className={ styles.icon }`), so we don't have to worry about namespacing the classes.

Reply:
Yes I've done that for the other components, but here I copied the extra rules over rather than doing that slightly bigger change. Definitely something that would be good to do as a follow up.